File: test-fixtures/language-service/swift/Sources/CommentDocLensSwift/OrderPresenter.swift
/// Formats the order status for customer-facing labels.
public func formatStatus(_ status: String) -> String {
    status.uppercased()
}

/// Presents order data in views.
public struct OrderPresenter {
    public init() {}

    public func label() -> String {
        formatStatus("paid")
    }
}

File: test-fixtures/language-service/csharp/OrderPresenter.cs
namespace CommentDocLens.CSharp;

public static class OrderPresenter
{
    /// <summary>
    /// Formats the order status for customer-facing labels.
    /// </summary>
    public static string FormatStatus(string status)
    {
        return status.ToUpperInvariant();
    }

    public static string BuildLabel(string status)
    {
        return FormatStatus(status);
    }
}

